### DISCUSSION TITLE
how do i change downloads drive on winget

### DISCUSSION THREAD

**Discussion by sastofficial** [microsoft/winget-pkgs#38686] [Q&A] [answered]
hi my c drive i really full and i wanna download my winget packages on my d drive instead. is that possible :)

**Comment by OfficialEsco** [comment, marked as answer]
You can choose the install location with `winget install <PackageName> -l` or `--location` IF the Package supports it.

**Reply by sastofficial** [reply]
is there a way to make my d drive the deafault drive

**Reply by OfficialEsco** [reply]
currently no, if you open `winget settings` you should have a $schema line which leads to https://aka.ms/winget-settings.schema.json
Here you can see all the available fields, the one you're looking for should be under InstallPrefReq
There is a issue for it already https://github.com/microsoft/winget-cli/issues/201
This is probably why its not added yet https://github.com/microsoft/winget-cli/issues/1639#issuecomment-950977736
